tests/test_nsjail.py

```diff
@@ -48,11 +48,10 @@ def test_memory_returns_137(self):
         self.assertEqual(result.stderr, None)
 
     def test_subprocess_resource_unavailable(self):
-        code = dedent("""
+        code = dedent(f"""
             import subprocess
 
-            # Max PIDs is 5.
-            for _ in range(6):
+            for _ in range({self.nsjail.config.cgroup_pids_max + 1}):
```

Comment:
I guess we could not include the `+ 1`, since the current process will count as 1, although it doesn't really matter.

Reply:
Indeed. I've removed this change in the recent rebase.